guides/common/assembly_preparing-for-disaster-recovery-and-recovering-from-data-loss.adoc

```diff
@@ -1,3 +1,5 @@
 include::modules/con_preparing-for-disaster-recovery-and-recovering-from-data-loss.adoc[]
 
 include::modules/con_overview-of-recommended-disaster-recovery-plans.adoc[leveloffset=+1]
+
+include::assembly_disaster-recovery-by-virtualizing-your-projectserver.adoc[leveloffset=+1]
```

Comment:
I'd prefer adding this assembly underneath `assembly_preparing-for-disaster-recovery-and-recovering-from-data-loss.adoc` in any `master.adoc` files. This makes it slightly easier to grasp/maintain. Downside: If you include the assembly twice, then you'll also have to include your new assembly twice. To me, this is a worthwhile tradeoff.

Reply:
I'd rather not, actually, because the way I'm thinking about this is that I'm composing `assembly_preparing-for-disaster-recovery-and-recovering-from-data-loss.adoc` as a whole. And `assembly_disaster-recovery-by-virtualizing-your-projectserver.adoc` is a part of that so I think it should be included in there. No change.